src/main/java/com/mindee/parsing/common/field/TaxField.java

```diff
@@ -3,14 +3,16 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.mindee.parsing.SummaryHelper;
+import java.util.HashMap;
 import lombok.Getter;
 
+
 /**
  * Represent a tax line.
  */
 @Getter
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class TaxField extends BaseField {
+public class TaxField extends LineItemField {
```

Comment:
If we change LineItemField to interface, it would look some thing like 
`public class TaxField extends extends BaseField implements LineItemField`